tests/files/secrets/project1.yml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: List
+items:
+- kind: ProjectRequest
+  apiVersion: v1
+  metadata:
+    name: oa-ci-secret1
+  displayName: OpenShift Applier Secret Test 1 (displayName)
```

Comment:
Not that it matters, but you've got some instances of `display_name`, and some of `displayName`

Reply:
Updated